test/test_performance.py.in

```diff
@@ -98,11 +114,12 @@ class PerformanceTestResults(unittest.TestCase):
     def test_results_@TEST_NAME@(self, performance_log_prefix, node_under_test, proc_info):
         self.addCleanup(_cleanUpLogs, performance_log_prefix)
 
-        launch_testing.asserts.assertExitCodes(
-            proc_info,
-            [launch_testing.asserts.EXIT_OK],
-            node_under_test,
-        )
+        # TODO(ahcorde): double free or corruption (fasttop) when FastRTPS finish
```

Comment:
Let's let the test fail and follow up with a proper solution. I've also seen this happen on my local machine but not repro on the buildfarm workers.

Reply:
In this case the test will not generate the data for this DDS implementation. Let's see what happens in the buildfarm.